central/telemetry/centralclient/instance_config.go

```diff
@@ -18,7 +20,8 @@ import (
 )
 
 const (
-	tenantIDLabel = "rhacs.redhat.com/tenant"
+	apiPathsAnnotation = "rhacs.redhat.com/telemetry-apipaths"
```

Comment:
I'd have the same question as in the previous PR within the series: whether we should move this to a more generic, importable location to re-use it within the fleetshard-synchronizer code.

Reply:
This one is really central specific. We've discussed this annotation with @rukletsov, and looks like we want to move it to an env var, as _we_ don't use annotations for configuration today.